packages/terra-brand-footer/examples/test-examples/DefaultBrandFooter.example.jsx

```diff
@@ -0,0 +1,51 @@
+          </svg>
+        </div>
+      </div>
+    }
+    contentBottom={
+      <p style={{ marignTop: '.7142857142857143rem' }} >
```

Comment:
The contentBottom prop should handle creating spacing here instead of using an inline style.